app/services/dashboard/participants.rb

```diff
@@ -101,10 +101,10 @@ def induction_records
                                       .where(induction_programmes: {
                                         school_cohort_id: dashboard_school_cohorts.map(&:id),
                                       }))
-                               .resolve
-                               .order("users.full_name")
-                               .inverse_induction_order
-                               .uniq(&:participant_profile_id)
+                                      .resolve
```

Comment:
The original indentation seems ok to me.

Reply:
I would be tempted to move the query into it's own method and reference it here to cut down on how messy this looks with all the indentation